pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -378,8 +382,14 @@ func (nim *nodeInfoManager) tryUpdateCSINode(
 	driverNodeID string,
 	maxAttachLimit int64,
 	topology map[string]string) error {
-
-	nodeInfo, err := csiKubeClient.StorageV1beta1().CSINodes().Get(string(nim.nodeName), metav1.GetOptions{})
+	var nodeInfo *storagev1beta1.CSINode
+	var err error
+	if nim.nodeLister != nil {
```

Comment:
I think we should require the informer always.  If informer is nil, then that means we're running in a standalone kubelet mode, which csi does not support. So I think we should skip initializing nodeinfomanager in that case.